integration-tests/src/main/java/org/apache/druid/testing/clients/OverlordResourceTestClient.java

```diff
@@ -127,7 +127,9 @@ public TaskStatusPlus getTaskStatus(String taskID)
               StringUtils.urlEncode(taskID)
           )
       );
-      LOG.debug("Index status response" + response.getContent());
+      if (LOG.isDebugEnabled()) {
```

Comment:
I suppose the changes in this class are not really needed as it is just a test class anyway. The changes wouldn't really have any impact.

Reply:
Okie, already reverted this one